resfulservice/src/controllers/curationController.js

```diff
@@ -428,3 +430,45 @@ const createBaseObject = (BaseObject, storedObject) => {
   }
   return curatedBaseObject;
 };
+
+const _getDoiInfo = async (req) => {
+  const { logger, params } = req;
+  logger.info('_getDoiInfo(): Function entry');
+  const { doi } = params;
+  const url = `${doiApi}${doi}`;
+  const headers = {
+    accept: 'application/json'
+  };
+
+  const preparedRequest = {
+    method: 'GET',
+    url,
+    headers
+  };
+  return await axios(preparedRequest);
+};
```

Comment:
Could we pass the preppedRequest to `_outboundRequest` for reusing existing code?

Reply:
So we can add a third argument to that function where if present, it should just make the call with the preppedRequest passed

Reply:
actually didn't need to add an argument for passing the prepped request, `_outboundRequest` works fine as is with just the uri passed